packages/syft/src/syft/node/node.py

```diff
@@ -879,22 +840,24 @@ def init_stores(
         self,
         document_store_config: StoreConfig | None = None,
         action_store_config: StoreConfig | None = None,
+        sqlite_path: Path | str | None = None,
     ) -> None:
+        # if there's no sqlite path, we'll use the tmp dir
+        if not sqlite_path:
```

Comment:
Yes it's actually used to place the database in a separate dir `$HOME/data/db` in `single_container_mode` so that we can mount volume on that dir.

Reply:
yes, but do we still need it as this level, can we not sort out that logic earlier and then just put it in the storconfig earlier? Because what would happen now if you would have both for instance? If we would have that logic higher in the chain of calls it might be easier to separate deployment logic from the logic in this node class

Reply:
agreed - i'll clean this up

Reply:
So i made the following changes:
- Completely removed this `sqlite_path`
- If you haven't provided either of action stores, it defaults to DictStoreConfig(). If `local_db=True` then uses a default SqliteStoreConfig
- Cleaned up init_store.
- Added default filename to SqliteStoreClientConfig to fix nonetype exceptions in single container mode